src/components/pages/ImageClassification/sampleData/image_endpoints.md

```
@@ -0,0 +1,1455 @@
+          "is_machine_created": true
+        }
+      ]
+    }
+  ]
+}
+```
+  - response: same as GET for individual image above, with modified annotations
+
+### delete image
+- example:
+  - DELETE https://dev-api.datamermaid.org/v1/projects/df2bb123-5bfc-4f3f-aa88-bb102d3b2b35/classification/images/103502bf-dfb7-4cb6-a13f-0b7094651f0d/
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)